packages/server/src/api/routes/tests/user.spec.js

```diff
@@ -23,19 +23,6 @@ describe("/users", () => {
   })
 
   describe("fetch", () => {
-    it("returns a list of users from an instance db", async () => {
```

Comment:
what was the reason for this being removed?

Reply:
I think it needs to be re-added again, i'm trying to fix a bug around this area and thought for a moment that because now uses have no access by default (you need specifically added to an app, we didn't need this test), but now i think that's unrelated, so I will add it back

Reply:
added back and test is functioning correctly.